README.md

```diff
@@ -11,16 +11,16 @@
 
 <p align="center">
   <strong>Guide → </strong>
-  <a href="#setup">Setup</a> ✯
-  <a href="#formats">Formats</a> ✯
-  <a href="#modern">Modern Mode</a> ✯
-  <a href="#usage">Usage &amp; Configuration</a> ✯
-  <a href="#options">All Options</a>
+  <a href="#-installation--setup--">Setup</a> ✯
```

Comment:
This doesn't appear to work on GitHub though

Reply:
GitHub's markdown renderer is notoriously inconsistent cross-platform, but this doesn't work for me on Windows Chrome & FF, Linux FF, or iOS Safari.

Test on the readme of your fork, does this link actually work? https://github.com/schalkventer/microbundle/tree/patch-1#-installation--setup--

Reply:
I can't find a single environment in which those trailing `-`'s are present -- what's your system details? Where are you running this where that gets generated?